transports/http/server/gin_loggers.go

```diff
@@ -0,0 +1,87 @@
+package httpserver
+
+import (
+	"time"
+
+	"github.com/bitcoin-sv/block-headers-service/config"
+	"github.com/gin-gonic/gin"
+	"github.com/rs/zerolog"
+)
+
+func setGinGlobals(cfg *config.HTTPConfig, log *zerolog.Logger) {
+	if cfg.ReleaseMode {
+		gin.SetMode(gin.ReleaseMode)
+	}
+
+	// Make GIN to use our logger for debugPrint, recovery messages and every other events when it uses fmt.Fprint(DefaultWriter/DefaultErrorWriter, ...)
+	// https://github.com/gin-gonic/gin/issues/1877#issuecomment-552637900
+	gin.DefaultWriter = newGinLogsWriter(log, ginDefaultWriterLevel(log))
+	gin.DefaultErrorWriter = newGinLogsWriter(log, zerolog.ErrorLevel)
+}
```

Comment:
I've realized that the `setGinGlobals` function might not be the best fit within the `gin_loggers.go` file, even though its primary purpose is setting global DefaultWriters. 

Let me know, and I'll explore ways to restructure this.

Reply:
IMHO, it would be nice to have this entire file in `/logging` directory, if it's possible, as it's all about logging

Reply:
I did as you suggested 👍

Reply:
I did as you suggested 👍